packages/aragon-wrapper/src/core/aragonOS/index.js

```diff
@@ -4,40 +4,72 @@ import { getAbi, getArtifact } from '../../interfaces'
 
 const aragonpmAppId = appName => namehash(`${appName}.aragonpm.eth`)
 
-const APP_MAPPINGS = new Map([
+const SYSTEM_APP_MAPPINGS = new Map([
   [aragonpmAppId('acl'), 'ACL'],
   [aragonpmAppId('evmreg'), 'EVM Script Registry'],
   [aragonpmAppId('kernel'), 'Kernel']
 ])
 
+const APM_APP_MAPPINGS = new Map([
+  [aragonpmAppId('apm-registry'), 'APM Registry'],
+  [aragonpmAppId('apm-repo'), 'Repo'],
+  [aragonpmAppId('apm-enssub'), 'ENS Subdomain Registrar']
```

Comment:
```suggestion
  [aragonpmAppId('apm-enssub'), 'ENS Subdomain Registrar'],
  [aragonpmAppId('apm-registry.open'), 'APM Registry'],
  [aragonpmAppId('apm-repo.open'), 'Repo'],
  [aragonpmAppId('apm-enssub.open'), 'ENS Subdomain Registrar']
```

See https://github.com/aragon/aragon-cli/pull/425#issuecomment-480488022

Reply:
Hmm, just hooked up the client to try and inspect an aragonPM instance and indeed we do need these overrides :/.

Not sure if we'll be able to do something clever but this might be the easiest way to get compatibility with the current `open.aragonpm.eth` deployment.

Reply:
I think this is a justifiable shortcut while we explore how to fix this properly so this contribution doesn't become stale.
I will open an issue and merge this PR with the related ones.

Reply:
These repos are actually set up by `APMRegistryFactory` [here](https://github.com/aragon/aragonOS/blob/dev/contracts/factory/APMRegistryFactory.sol#L76-L94) and the appIds are always set to the `apm-*` repos in the registry. 

All registries that are created with this factory will have the same issue. The factory was developed for deploying the root registry and we didn't take this into account.